package.json

```diff
@@ -8,6 +8,7 @@
   ],
   "scripts": {
     "build": "stencil build --prerender",
+    "build2": "node build.js",
```

Comment:
just a suggestion about naming, instead of build2, looks better build:node for example, to indicate is a build with node